CHANGELOG.md

```diff
@@ -1,19 +1,29 @@
-## 10.1.3 (YYYY-MM-DD)
+## 10.2.0 (2020-12-02)
+
+### Deprecated
+* [RealmApp] `Credentials.google(authenticationCode: String)`. Use `Credentials.google(token: String, authType: GoogleAuthType)` instead.
 
 ### Breaking Changes
 * None.
 
 ### Enhancements
-* None.
+* [RealmApp] Added `Credentials.google(token: String, authType: GoogleAuthType)`, as MongoDB Realm support multiple ways of logging in Google Accounts.
 
 ### Fixes
+* [RealmApp] Bug that would prevent eventual consistency during conflict resolution. Affected clients would experience data divergence and potentially consistency errors as a result if they experienced conflict resolution between cycles of Create-Erase-Create for objects with primary keys.
```

Comment:
A bit convoluted. Tried to rephrase but failed as I was a bit worried that I changed the meaning. Maybe just re-read to verify.

Reply:
This is a copy from the Sync changelog. I agree it is a bit convoluted, but so was the error, so not sure there is much we can do to rephrase it 🤔

Reply:
Heh. I didn't even get that this was actually the _protocol 2_ snippet.